mock/mock.go

```diff
@@ -187,6 +191,19 @@ func (c *Call) Run(fn func(args Arguments)) *Call {
 	return c
 }
 
+// ReturnFn sets a handler to be called before returning.
+//
+//	Mock.On("MyMethod", arg1, arg2).ReturnFn(func(args Arguments) Arguments {
+//	    return Arguments{args.Get(0) + args.Get(1)}
```

Comment:
The example should be valid Go, did you mean Int rather than Get?
```suggestion
//	    return Arguments{args.Int(0) + args.Int(1)}
```